content/docs/api-reference/dvcfilesystem.md

```diff
@@ -0,0 +1,126 @@
+# DVCFileSystem
+
+_New in DVC 2.27_
+
+DVCFileSystem provides a pythonic file interface (
+[fsspec-compatible](https://filesystem-spec.readthedocs.io/)) for a DVC repo. It
+is a read-only filesystem, hence it does not support any write operations, like
+`put_file`, `cp`, `rm`, `mv`, `mkdir` etc.
+
+DVCFileSystem provides a unified view of all the files/directories in your
+repository, be it Git-tracked or DVC-tracked, or untracked (in case of a local
+repository). It can reuse the files in DVC <abbr>cache</abbr> and can otherwise
+stream from
+[supported remote storage](/doc/command-reference/remote/add#supported-storage-types).
```

Comment:
```suggestion
`DVCFileSystem` objects construct a unified view of all the files and directories in your
repo, be it Git-tracked, DVC-tracked, or untracked (in the case of local
repos). It can reuse files in the <abbr>DVC cache</abbr>, and it can otherwise
stream from [supported remote storage].

[supported remote storage]: /doc/command-reference/remote/add#supported-storage-types
```

> It can reuse files in the DVC cache

What does this mean though? Reuse them for what purpose?